compiler/src/iree/compiler/Dialect/TensorExt/IR/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+################################################################################
+# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
+# compiler/src/iree/compiler/Dialect/TensorExt/IR/BUILD.bazel                  #
+#                                                                              #
+# Use iree_cmake_extra_content from iree/build_defs.oss.bzl to add arbitrary   #
+# CMake-only content.                                                          #
+#                                                                              #
+# To disable autogeneration for this file entirely, delete this header.        #
+################################################################################
+
+iree_add_all_subdirs()
+
+iree_cc_library(
+  NAME
+    IR
+  HDRS
+    "TensorExtDialect.h"
+    "TensorExtDialect.h.inc"
+    "TensorExtOps.h"
+    "TensorExtOps.h.inc"
+    "TensorExtTypes.h"
+  SRCS
+    "TensorExtDialect.cpp"
+    "TensorExtDialect.cpp.inc"
+    "TensorExtOpFolders.cpp"
+    "TensorExtOps.cpp"
+    "TensorExtOps.cpp.inc"
+    "TensorExtTypes.cpp"
+  DEPS
+    ::TensorExtOpsGen
+    LLVMSupport
+    MLIRArithDialect
+    MLIRArithUtils
+    MLIRIR
+    MLIRInferTypeOpInterface
+    MLIRTensorDialect
+    iree::compiler::Dialect::Encoding::IR
+    iree::compiler::Dialect::Util::IR
+  PUBLIC
+)
+
+iree_tablegen_library(
+  NAME
+    TensorExtOpsGen
+  TD_FILE
+    "TensorExtOps.td"
+  OUTS
+    --gen-op-decls TensorExtOps.h.inc
+    --gen-op-defs TensorExtOps.cpp.inc
+    --dialect=iree_tensor_ext --gen-dialect-decls TensorExtDialect.h.inc
+    --dialect=iree_tensor_ext --gen-dialect-defs TensorExtDialect.cpp.inc
+)
+
+iree_tablegen_doc(
+  NAME
+    TensorExtDialectDocGen
+  TD_FILE
+    "TensorExtOps.td"
+  OUTS
+    --gen-dialect-doc -dialect=iree_tensor_ext TensorExtDialect.md
+)
```

Comment:
For new dialect docs to be included on the website (https://iree.dev/reference/mlir-dialects/), you'll want to edit a few more files under `docs/`:

* https://github.com/iree-org/iree/blob/main/docs/website/postprocess_dialect_docs.py
* https://github.com/iree-org/iree/blob/main/docs/website/docs/reference/mlir-dialects/index.md

Docs for how to test such changes: https://github.com/iree-org/iree/tree/main/docs/website#how-to-edit-this-documentation

Reply:
Thanks for the pointers, I created a PR for this: https://github.com/iree-org/iree/pull/20669